### DISCUSSION TITLE
Welcome to Machine Learning Projects 101 Discussions!

### DISCUSSION THREAD

**Discussion by muhammadanas0716** [muhammadanas0716/Machine-Learning-101#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Runner3131** [comment]
Hey, My name is Timothy. and I'm a Python programmer mainly into AI and machine learning. also web development. 
Anyway, I wondered if we could add AI exercises to the repo, since AI and machine learning go hand in hand. just a thought.

**Reply by muhammadanas0716** [reply]
Absolutely. An excellent idea. If you could actually take this responsibility to do this, I'd be ever so grateful. Otherwise I'll do it over the weekend. Let me know your thoughts.
